src/ResourceManager/ApiManagement/Commands.ApiManagement.ServiceManagement/Commands/PublishAzureApiManagementTenantConfiguration.cs

```diff
@@ -0,0 +1,81 @@
+﻿//  
+// Copyright (c) Microsoft.  All rights reserved.
+// 
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+// 
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+namespace Microsoft.Azure.Commands.ApiManagement.ServiceManagement.Commands
+{
+    using System;
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.ApiManagement.ServiceManagement.Models;
+
+    [Cmdlet(VerbsData.Publish, Constants.ApiManagementTenantGitConfiguration)]
+    [OutputType(typeof(PsApiManagementOperationResult))]
+    public class PublishAzureApiManagementTenantConfiguration : AzureApiManagementCmdletBase
+    {
+        [Parameter(
+            ValueFromPipelineByPropertyName = true,
+            Mandatory = true,
+            HelpMessage = "Instance of PsApiManagementContext. This parameter is required.")]
+        [ValidateNotNullOrEmpty]
+        public PsApiManagementContext Context { get; set; }
+
+        [Parameter(
+            ValueFromPipelineByPropertyName = true,
+            Mandatory = true,
+            HelpMessage = "Name of the Git branch from which the configuration is to be deployed to the configuration database. This parameter is required.")]
+        [ValidateNotNullOrEmpty]
+        public String Branch { get; set; }
+
+        [Parameter(
+            ValueFromPipelineByPropertyName = true,
+            Mandatory = false,
+            HelpMessage = "Enforce deleting subscriptions to products that are deleted in this update. This parameter is optional. Default value is false.")]
+        public bool? Force { get; set; }
```

Comment:
Force should be reserved for a SwitchParameter that short-circuits a call to ShouldContinue.  If this parameter is for some other purpose, you should call it something else.  And it should be a swtch parameter, regardless.